test/utils.string.test.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
Same with this file, I'd probably keep this one indented with 2 spaces consistently.